components/Modal/Schedule/ScheduleCreate/index.tsx

```diff
@@ -1 +1,222 @@
+              <Image src="/icons/location-icon.svg" alt="장소" width={20} height={20} />
+              <input
+                className={cn("place-input")}
+                type="text"
+                maxLength={19}
+                placeholder="(선택) 위워크 10층 미팅룸"
+                {...register("location")}
+              />
+              <p className={cn("location-length")}>
+                <b>{locationWatch.length} </b> / 20
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className={cn("description")}>
+          <p className={cn("label")}>추가 내용</p>
+          <textarea
+            className={cn("text")}
+            maxLength={79}
+            placeholder="(선택) 설명을 입력해주세요"
+            {...register("description")}
+          />
+          <p className={cn("text-length")}>
+            <b>{descriptionWatch.length}</b> / 80
+          </p>
+        </div>
+        <div className={cn("group")}>
+          <p className={cn("label")}>그룹 선택</p>
+          <div
+            className={cn("group-select")}
+            onClick={() => setIsGroupListVisible(!isGroupListVisible)}
+          >
+            {selectedGroup ? (
+              <div className={cn("selected-group")}>
+                <div
+                  className={cn("group-color")}
+                  style={{ backgroundColor: selectedGroup.labelColor }}
+                />
+                <p className={cn("group-name")}>{selectedGroup.crewName}</p>
+              </div>
+            ) : (
+              <p className={cn("placeholder")}>그룹을 선택하세요</p>
+            )}
+          </div>
+          {isGroupListVisible && (
+            <div className={cn("group-container")}>
+              {groupList ? (
+                groupList.map((group) => (
+                  <div
+                    key={group.crewId}
+                    className={cn("group-list")}
+                    onClick={() => handleGroupSelect(group)}
+                  >
+                    <div
+                      className={cn("group-color")}
+                      style={{ backgroundColor: `${group.labelColor}` }}
+                    />
+                    <p className={cn("group-name")}>{group.crewName}</p>
+                  </div>
+                ))
+              ) : (
+                <div>그룹을 먼저 만들어주세요</div>
+              )}
+            </div>
+          )}
+        </div>
+        <div className={cn("member-container")}>
+          <p className={cn("label")}>참가자 선택</p>
+          <div className={cn("members")}>
+            {groupInfo?.crewMembers.length === 1 ? (
+              <p className={cn("alone-msg")}>그룹에 팀원이 없습니다.</p>
+            ) : (
+              groupInfo?.crewMembers.map(
+                (member) =>
+                  member.accountId !== userInfo?.accountId && (
+                    <React.Fragment key={member.accountId}>
+                      <input
+                        id={`member-${member.accountId}`}
```

Comment:
```suggestion
                        id={member.accountId}
```
P4: `member.accountId`이렇게만 적었을 때 생기는 문제가 있을까요??

Reply:
number 형식은 string혛식에 넣을 수 없다고 나와요!